src/test/java/org/broadinstitute/hellbender/utils/bwa/BwaMemIndexTest.java

```diff
@@ -1,19 +1,29 @@
 package org.broadinstitute.hellbender.utils.bwa;
 
 import org.testng.Assert;
+import org.testng.Reporter;
 import org.testng.annotations.AfterClass;
 import org.testng.annotations.BeforeClass;
+import org.testng.annotations.DataProvider;
 import org.testng.annotations.Test;
 
 import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.PrintWriter;
 import java.util.ArrayList;
 import java.util.Collections;
 import java.util.List;
+import java.util.Random;
+import java.util.*;
+import java.util.stream.Stream;
 
 @Test
 public final class BwaMemIndexTest {
```

Comment:
Perhaps the test would be to  compare a set of alignments produced by both indexes, these alignments could cover the whole small reference in a systematic manner. @tedsharpe do you think that that would be deterministic enough?